docs/guides/error-mitigation-and-suppression-techniques.ipynb

```diff
@@ -120,6 +120,74 @@
     "estimator.options.twirling.shots_per_randomization = 100"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "e5dcab7b",
+   "metadata": {},
+   "source": [
+    "## Probabilistic Error Amplification (PEA)\n",
+    "\n",
+    "PEA is an aplification technique used with [Zero-noise extrapolation(ZNE)](#zne).  The main challenge in successfully implmenting ZNE is to have an accurate model for noise in the expectation value and to amplify the noise in a known way.  For utility-scale experiments, PEA is often the best choice because it can be applied to any circuit that can be run with native noise factor $(\\lambda - 1)$. \n",
+    "\n",
+    "Because PEA is a ZNE amplification technique, you also need to enable ZNE by setting `resilience.zne_mitigation = True`. Other `resilience.zne` options can be used in addition to set extrapolators, amplification levels, and so on. PEA requries a noise model, which is automatically generated when using primitives. \n",
+    "\n",
+    "When the Estimator primitive is used to call `noise-learner`, PEA can reuse the returned noise model. **LINK TO NOISE LEARNER DOCS** "
```

Comment:
Here, you are not using the noise learner. I would simply phrase things as follows:

The following snippet provides an example where PEA is used to mitigate the result of an estimator job:
```
estimator = Estimator(mode=backend)
estimator.options.resilience.zne_mitigation = True
estimator.options.resilience.zne.amplifier = "pea"

estimator.run(pubs, precision=0.01)
```